packages/node_modules/@ciscospark/widget-space/src/selector.js

```diff
@@ -69,14 +56,52 @@ export const getActivityTypes = createSelector(
   }
 );
 
+const getWidgetError = createSelector(
+  [getWidget, getSpark, getFormatMessage],
+  (widget, spark, formatMessage) => { // eslint-disable-line complexity
+    const registerError = spark.getIn([`status`, `registerError`]);
+    const widgetError = widget.getIn([`status`, `isError`]);
```

Comment:
Yea, I don't think it was this code, but we definitely wanted to use `hasError`. I'll change that.